apps/web/src/components/Basenames/RegistrationContext.tsx

```diff
@@ -178,23 +178,15 @@ export default function RegistrationProvider({ children }: RegistrationProviderP
         });
       }
     }
-    if (callsIsSuccess && callsData) {
-      const successCall = callsData.receipts?.find((receipt) => receipt.status === 'success');
-      if (successCall) {
-        logEventWithContext('register_name_transaction_success', ActionType.change);
-        baseEnsNameRefetch()
-          .then(() => setRegistrationStep(RegistrationSteps.Success))
-          .catch(() => {});
-      } else {
-        const failCall = callsData.receipts?.find((receipt) => receipt.status !== 'success');
-        logEventWithContext('register_name_transaction_reverted', ActionType.change, {
-          error: `Smart wallet transaction reverted: ${failCall?.transactionHash}`,
-        });
-      }
+    if ((callsIsSuccess && callsData) || callsError) {
+      baseEnsNameRefetch()
```

Comment:
Is there any chance of a race condition here? I'm wondering whether we should pull out to a separate useEffect that just watches `useBaseEnsName` and then advances to the Success step, and put `useBaseEnsName` on a short refresh interval